examples/cs/interop-non-transpilable-code/output/index.cs

```diff
@@ -0,0 +1,19 @@
+using Helpers;
+
+class MyClass
+{
+    public virtual void mainFeature(object message)
+    {
+        object ar = new List<object>() {1, 2, 3};
```

Comment:
@carlosmiei here  `object ar` - how much effort needed so we turned that into either autodetected (so `List<object>`) or turned it to `var` instead of object?

Reply:
that's easy the problem is in the ccxt side